### ISSUE TITLE
voice over: improve message send button

### ISSUE THREAD

**Issue by cyBerta** [deltachat/deltachat-ios#1535]
Currently you can swipe in a chat through the message input bar UI elements using VoiceOver. There is no way for the user to distinguish the enabeld from the disabled state of the send button when using VoiceOver. The state of the button should be communicated so that it is easier to recognize that no action is available.
